iceoryx_examples/iceperf/main_follower.cpp

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#include "iceperf_follower.hpp"
+
+#include "iceoryx_utils/platform/getopt.hpp"
+
+#include <iostream>
+
+int main(int argc, char* argv[])
+{
+    constexpr option longOptions[] = {{"help", no_argument, nullptr, 'h'}, {nullptr, 0, nullptr, 0}};
+
+    // colon after shortOption means it requires an argument, two colons mean optional argument
+    constexpr const char* shortOptions = "h";
+    int32_t index{0};
+    int32_t opt{-1};
+    while ((opt = getopt_long(argc, argv, shortOptions, longOptions, &index), opt != -1))
+    {
+        switch (opt)
+        {
+        case 'h':
+            std::cout << "Usage: " << argv[0] << " [options]" << std::endl;
+            std::cout << "Options:" << std::endl;
+            std::cout << "-h, --help                        Display help" << std::endl;
```

Comment:
I thought there were more options? You promised an exciting list of options in the readme and now I realize the only options `-h`?! To inform the user that there is the option `-h` ...

Please add an additional exciting and fascinating option! https://www.youtube.com/watch?v=I57hkclh3kk